common/app/views/fragments/javaScriptLaterSteps.scala.html

```diff
@@ -301,7 +301,13 @@
                     images.upgradePictures();
                     images.listen();
 
-                    if (config.switches.commercial) {
+                    // Preference pages are served via HTTPS for service worker support.
+                    // These pages must not have mixed (HTTP/HTTPS) content, so
+                    // we disable ads (until the day comes when all ads are HTTPS).
+                    // TODO: prefs "section"?
+                    var isPreferencePage = config.page.pageId === 'preferences'
```

Comment:
we can override the `metaData` map in `PreferencesMetaData` to pass config to the page. ill add it now